sites/offshore-mag/server/templates/index.marko

```diff
@@ -34,26 +34,111 @@ $ const adSlots = {
 
   <div class="row">
     <div class="col-lg-8">
-      <endeavor-content-query-feed
-        header="Featured"
-        limit=7
-        skip=5
-        section-id=section.id
-        native-x={ placement: 'list2', aliases, index: 6 }
-      />
+      <div class="row">
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="regional-reports"
+            header={ title: "Regional Reports", href: "/regional-reports" }
+            native-x={ placement: 'list1', aliases: ['regional-reports'], index: 3 }
```

Comment:
This will fallback to default because `regional-reports` is not a valid NativeX topic (per instructions yesterday)